README.md

```diff
@@ -50,6 +50,13 @@ yum install krb5-libs
 brew install krb5
 ```
 
+#### WIndows
+```bash
+Down and install the Microsoft Kerberos package from https://web.mit.edu/kerberos/dist/
+
+Copy the <INSTALL FOLDER>\MIT\Kerberos\bin\gssapi64.dll to then folder in %PATH% ,and change the name to gssapi_krb5.dll
+```
```

Comment:
```suggestion
Download and install the Microsoft Kerberos package from https://web.mit.edu/kerberos/dist/

Copy the `<INSTALL FOLDER>\MIT\Kerberos\bin\gssapi64.dll` file to a folder in %PATH% and change the name to `gssapi_krb5.dll`
```
Do you know if this gets installed into your path automatically? Is it just the file name that is the issue?

Reply:
> Do you know if this gets installed into your path automatically? Is it just the file name that is the issue?

It 's  won't  get installed into my path automatically